packages/database/src/analytics/AnalyticsRefresher.js

```diff
@@ -57,6 +59,13 @@ export class AnalyticsRefresher {
   };
 
   static async executeRefresh(database) {
-    await database.executeSql(`SELECT mv$refreshMaterializedView('analytics', 'public', true);`);
+    try {
+      const start = Date.now();
+      await database.executeSql(`SELECT mv$refreshMaterializedView('analytics', 'public', true);`);
+      const end = Date.now();
+      winston.info(`Analytics table refresh took: ${end - start}ms`);
```

Comment:
This could potentially end up rather noisy, however it may also help us keep track analytics refreshes and how long they are ending up taking. I'm leaning towards keeping it but happy to hear other opinions